baseplate/experiments/providers/base.py

```diff
@@ -0,0 +1,24 @@
+
```

Comment:
nit: please make sure all the new files have the `from __future__` stuff at the top